core/storage/user/gae_models_test.py

```diff
@@ -194,80 +257,81 @@ def test_instance_stores_correct_data(self):
-            has_not_logged_in_for_n_days=has_not_logged_in_for_n_days,
-            submitter_id=submitter_id).put()
-
-        submitter_id = 'submitter_2'
-        query_id = 'qid_2'
-        inactive_in_last_n_days = 6
-        created_at_least_n_exps = 7
-        created_fewer_than_n_exps = 4
-        edited_at_least_n_exps = 3
-        edited_fewer_than_n_exps = 6
-        has_not_logged_in_for_n_days = 11
-        user_models.UserQueryModel(
-            id=query_id,
-            inactive_in_last_n_days=inactive_in_last_n_days,
-            created_at_least_n_exps=created_at_least_n_exps,
-            created_fewer_than_n_exps=created_fewer_than_n_exps,
-            edited_at_least_n_exps=edited_at_least_n_exps,
-            edited_fewer_than_n_exps=edited_fewer_than_n_exps,
-            has_not_logged_in_for_n_days=has_not_logged_in_for_n_days,
-            submitter_id=submitter_id).put()
-
-        # Fetch only one entity.
-        query_models, _, _ = user_models.UserQueryModel.fetch_page(
-            1, None)
-        self.assertEqual(len(query_models), 1)
-
-        self.assertEqual(query_models[0].submitter_id, 'submitter_2')
-        self.assertEqual(query_models[0].id, 'qid_2')
-        self.assertEqual(query_models[0].inactive_in_last_n_days, 6)
-        self.assertEqual(query_models[0].created_at_least_n_exps, 7)
-        self.assertEqual(query_models[0].created_fewer_than_n_exps, 4)
-        self.assertEqual(query_models[0].edited_at_least_n_exps, 3)
-        self.assertEqual(query_models[0].edited_fewer_than_n_exps, 6)
-        self.assertEqual(query_models[0].has_not_logged_in_for_n_days, 11)
-
-        # Fetch both entities.
-        query_models, _, _ = user_models.UserQueryModel.fetch_page(
-            2, None)
-        self.assertEqual(len(query_models), 2)
-
-        self.assertEqual(query_models[0].submitter_id, 'submitter_2')
-        self.assertEqual(query_models[0].id, 'qid_2')
-        self.assertEqual(query_models[0].inactive_in_last_n_days, 6)
-        self.assertEqual(query_models[0].created_at_least_n_exps, 7)
-        self.assertEqual(query_models[0].created_fewer_than_n_exps, 4)
-        self.assertEqual(query_models[0].edited_at_least_n_exps, 3)
-        self.assertEqual(query_models[0].edited_fewer_than_n_exps, 6)
-        self.assertEqual(query_models[0].has_not_logged_in_for_n_days, 11)
-
-        self.assertEqual(query_models[1].submitter_id, 'submitter_1')
-        self.assertEqual(query_models[1].id, 'qid_1')
-        self.assertEqual(query_models[1].inactive_in_last_n_days, 5)
-        self.assertEqual(query_models[1].created_at_least_n_exps, 1)
-        self.assertEqual(query_models[1].created_fewer_than_n_exps, 3)
-        self.assertEqual(query_models[1].edited_at_least_n_exps, 2)
-        self.assertEqual(query_models[1].edited_fewer_than_n_exps, 5)
-        self.assertEqual(query_models[1].has_not_logged_in_for_n_days, 10)
+def test_fetch_page(self):
+
+    submitter_id = 'submitter_1'
```

Comment:
I don't think you're indenting this correctly. See the diff of the PR -- please look at the Files Changed tab.

Reply:
@ulloaluis

Reply:
Not sure what this relates to, I didn't do anything with that code?

Reply:
Please take a look at the "Files Changed" tab and ensure that the diff is what you expect it to be.

Reply:
This seems to reference the test_fetch_page function spacing, but I don't think I'm the one that added that deleted/added that?

Reply:
Ah yep, I don't know who did it, but it needs to be fixed (by someone) before this PR can be merged.

Reply:
Done